freqtrade/rpc/telegram.py

```diff
@@ -1981,16 +1981,18 @@ async def _list_custom_data(self, update: Update, context: CallbackContext) -> N
             results = self._rpc._rpc_list_custom_data(trade_id, key)
             messages = []
             if len(results) > 0:
-                messages.append("Found custom-data entr" + ("ies: " if len(results) > 1 else "y: "))
-                for result in results:
+                trade_custom_data = results[0]["custom_data"]
+                messages.append(
+                    "Found custom-data entr" + ("ies: " if len(trade_custom_data) > 1 else "y: ")
+                )
+                for custom_data in trade_custom_data:
                     lines = [
-                        f"*Key:* `{result['cd_key']}`",
-                        f"*ID:* `{result['id']}`",
-                        f"*Trade ID:* `{result['ft_trade_id']}`",
-                        f"*Type:* `{result['cd_type']}`",
-                        f"*Value:* `{result['cd_value']}`",
-                        f"*Create Date:* `{format_date(result['created_at'])}`",
-                        f"*Update Date:* `{format_date(result['updated_at'])}`",
+                        f"*Trade ID:* `{trade_id}`",
```

Comment:
I don't think tradeId should be in the output.

the telegram command is quite explicit - and only allows `/list_custom_data <tradeid> [key]` - if tradeID is no integer, it'll fail - so it unnecessarily extends the message.